modules/gitops-enabling-the-creation-of-aggregated-cluster-roles.adoc

```diff
@@ -0,0 +1,270 @@
+[id="gitops-enabling-the-creation-of-aggregated-cluster-roles_{context}"]
+= Enabling the creation of aggregated cluster roles
+
+To enable the creation of aggregated cluster roles for the Argo CD Application Controller component of a cluster-scoped Argo CD instance, you must configure the corresponding field by editing the YAML file of the Argo CD custom resource (CR).
+
+.Procedure
+
+. In the Argo CD CR, set the value of the `.spec.aggregatedClusterRoles` field to `true`:
++
+.Example Argo CD CR
+[source,yaml]
+----
+apiVersion: argoproj.io/v1beta1
+kind: ArgoCD
+metadata:
+  name: example # <1>
+  namespace: spring-petclinic # <2>
+# ...
+spec:
+  aggregatedClusterRoles: true # <3>
+# ...
+----
+<1> The name of the cluster-scoped instance.
+<2> The namespace where you want to run the cluster-scoped instance.
+<3> The value set to `true` enables the creation of aggregated cluster roles. If you do not want to enable the creation of aggregated cluster roles, do not include this line or set the value to `false`.
++
+.Example output
+[source,terminal]
+----
+argocd.argoproj.io/example configured
+----
+
+. Verify that the `Status` field of the cluster-scoped Argo CD instance shows as `Phase: Available` by running the following command:
++
+[source,terminal]
+----
+$ oc describe argocd.argoproj.io/example -n spring-petclinic
+----
++
+.Example output
+[source,terminal]
+----
+Name:         example
+Namespace:    spring-petclinic
+Labels:       <none>
+Annotations:  <none>
+API Version:  argoproj.io/v1beta1
+Kind:         ArgoCD
+Metadata:
+  Creation Timestamp:  2024-08-14T08:20:53Z
+  Finalizers:
+    argoproj.io/finalizer
+  Generation:        3
+  Resource Version:  60437
+  UID:               57940e54-d60b-4c1a-bc4a-85c81c63ab69
+Spec:
+  Aggregated Cluster Roles:  true
+  Controller:
+    Processors:
+    Sharding:
+  Grafana:
+    Enabled:  false
+    Ingress:
+      Enabled:  false
+    Route:
+      Enabled:  false
+  Ha:
+    Enabled:  false
+  Initial SSH Known Hosts:
+  Monitoring:
+    Enabled:  false
+  Notifications:
+    Enabled:  false
+  Prometheus:
+    Enabled:  false
+    Ingress:
+      Enabled:  false
+    Route:
+      Enabled:  false
+  Rbac:
+  Redis:
+  Repo:
+  Server:
+    Autoscale:
+      Enabled:  false
+    Grpc:
+      Ingress:
+        Enabled:  false
+    Ingress:
+      Enabled:  false
+    Route:
+      Enabled:  false
+    Service:
+      Type:  
+  Tls:
+    Ca:
+Status:
+  Application Controller:      Running
+  Application Set Controller:  Unknown
+  Phase:                       Available # <1>
+  Redis:                       Running
+  Repo:                        Running
+  Server:                      Running
+  Sso:                         Unknown
+Events:                        <none>
+----
+<1> The status indicates that the cluster-scoped Argo CD instance is healthy and available.
++
+The {gitops-title} Operator creates the following default cluster roles and manages them:
```

Comment:
```suggestion
[NOTE]
====
The {gitops-title} Operator creates the following default cluster roles and manages them:

* `<argocd_name>-<argocd_namespace>-argocd-application-controller` aggregated cluster role
* `<argocd_name>-<argocd_namespace>-argocd-application-controller-view`
* `<argocd_name>-<argocd_namespace>-argocd-application-controller-admin` 
====
```
I think this info should be a NOTE because it isn't part of the procedure.

Reply:
Addressed